docs/project/SupportedCloudProviders.rst

```diff
@@ -3,45 +3,49 @@
++-----------------------+-----------------------------------------------+-------------------------------------------------------------------------------------------------+-------------------------------------------------------------+
+
 .. note::
 
    The *nRF Asset Tracker* is not a *multi-cloud* solution.
    The concrete implementation will differ for each cloud provider.
-   It does not aim to provide a solution where you can switch in a seamlessmanner between cloud providers.
-   Such a solution will require the implementation of additional abstraction layers, which instead of becoming a meaningful benefit for the majority of users, mandate a constant translation between the vendor specific concepts and the abstraction.
-   Instead, the best practices of the respective cloud-vendor for building a *serverless* IoT solution are followed.
-
-See the below table for a list of supported cloud providers:
+   This solution does not aim to provide a solution where you can switch in a seamless manner between cloud providers.
 
-+-------------------+-----------------------------------------------+-------------------------------------------------------------------------------------------------+-------------------------------------------------------------+
-|                   | Amazon Web Services                           | Microsoft Azure                                                                                 | Google Cloud                                                |
-+===================+===============================================+=================================================================================================+=============================================================+
-| LTE-M: TCP+MQTT   | :ref:`feature complete <aws-getting-started>` | `in progress <https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/discussions/12>`_ | `on hold <https://github.com/bifravst/bifravst/issues/25>`_ |
-+-------------------+-----------------------------------------------+-------------------------------------------------------------------------------------------------+-------------------------------------------------------------+
-| NB-IoT: UDP+LWM2M | in consideration                              | in consideration                                                                                |                                                             |
-+-------------------+-----------------------------------------------+-------------------------------------------------------------------------------------------------+-------------------------------------------------------------+
+   
+A multi-cloud solution with a functionality of switching between the clouds will require the implementation of additional abstraction layers.
```

Comment:
This refers to "not being a multi-cloud solution" which is mentioned in the Note above, but it's outside of the note. It should be in the note, because it extends on it.